src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,6 +142,16 @@ public static void main(final String[] args) {
     ErrorConsole.getConsole();
     // do after we handle command line args
     checkForMemoryXMX();
+    Toolkit.getDefaultToolkit().getSystemEventQueue().push(new EventQueue() {
```

Comment:
In swing every method passes the eventqueue at some time right? We are basically overrdiding the default eventqueue to log every throwable which may be thrown by `super.dispatchEvent`. See [this Page](https://ruben42.wordpress.com/2009/03/30/catching-all-runtime-exceptions-in-swing/) for more information.